examples/OECC/README.md

```diff
@@ -0,0 +1,143 @@
+# OpenEdge Command Center #
+
+Demo scripts to setup an OpenEdge Command Center environment with support for OpenTelemetry metrics.
+
+The scripts can be run on a virtual machine running Ubuntu Linux 22.04.
+The VM could be created on AWS, Azure or other virtualization environment.
+Scripts run commands to detect the IP address of the machine.
+
+### Requirements 
+* Ubuntu Linux 22.04
+    * user with sudo access
+* OpenEdge media:
+    * PROGRESS_OE_12.8.3_LNX_64.tar.gz
+    * PROGRESS_OECC_SERVER_1.3.0_LNX_64.tar.gz
```

Comment:
It may be nice to point out where to find these in the Progress ESD site. I honestly had a difficult time trying to find the OECC installation files.

Reply:
Added note regarding the ESD.